### ISSUE TITLE
update publication pipeline

### ISSUE THREAD

**Issue by keighrim** [clamsproject/mmif#100]
When py-0.1.0 was published as `mmif-python` on pypi, the binary distribution was wrongly uploaded as source distribution. As a result, `pip install mmif-python` fails. Make sure this doesn't happen in the future .

**Comment by keighrim** [comment]
67bedcfbc58b6f78b5a80bab733abd7a62d1fcbe fixes all issues with `pip install mmif-python`. Verifiable by installing a dev version from in-house pypi via 
```
pip install --extra-index-url http://morbius.cs-i.brandeis.edu:8081/repository/pypi-develop/simple mmif-python --pre
```
.

**Comment by keighrim** [comment]
Closed via #108 .
